docs/notebooks/expected_improvement.pct.py

```diff
@@ -201,6 +201,54 @@ def build_model(data):
     idx_best=arg_min_idx,
 )
 
+# %% [markdown]
+# ## Batch-sequential strategy
+#
+# Sometimes it is practically convenient to query several points at a time. We can do this in `trieste` using `BatchMonteCarloExpectedImprovement`. Note that this acquisition is computed using a Monte-Carlo method (so it requires a `sample_size`), but with a reparametrisation trick, which makes it deterministic. To use it, we change the acquisition rule and specify the number of query points (`num_query_points`) to be chosen, then observed simultaneously:
```

Comment:
when you say

> it is practically convenient to query several points at a time

it might be helpful to explain more about what's going on here: that we use the batch acquisition rule which recommends a number of points, and a batch acquisition function such as batch EI (we don't have to use batch EI for batch BO). This optimizer then queries the observer at all these points. Some of this is duplicating the optimizer docs so I don't know how much you think we should include.

Reply:
also could you explain "practically convenient"? I'm not sure how to improve it, but it's a little vague

Reply:
I meant that in some cases you can query several points at a time with little extra cost (for instance if you can parallelize the calls to an expensive observer).

Reply:
is it worth adding that? or will it be obvious to most readers?

Reply:
I think this is obvious